src/helpers/file.ts

```diff
@@ -0,0 +1,7 @@
+import * as shell from 'shelljs';
+
+export const readConfigFile = () => {
+  const { DOBBY_CONFIG } = shell.env;
```

Comment:
Why are you using { } ?

Reply:
It's called destructuring assignment, when you assign a variable with the same name of the origin on the other object.